core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -659,6 +650,33 @@ public String send(MessageProducer messageProducer, javax.jms.Message message, b
 		}
 	}
 
+	protected void logMessageDetails(javax.jms.Message message, MessageProducer messageProducer) throws JMSException {
+		if (log.isInfoEnabled() && !log.isDebugEnabled()) {
```

Comment:
```suggestion
		if (log.isInfoEnabled()) {
```
Beetje dubbel op dit :)

Reply:
Duidelijk!